### ISSUE TITLE
arangorestore unable to create index

### ISSUE THREAD

**Issue by ranjithk-redbus** [arangodb/arangodb#6584]
## My Environment

* __ArangoDB Version__:        3.3.4 
* __Storage Engine__:             
* __Deployment Mode__:       Single Server 
* __Deployment Strategy__:   
* __Configuration__:               
* __Infrastructure__:               
* __Operating System__:        
* __Used Package__:              

## Component, Query & Data

__Affected feature__:

__AQL query (if applicable)__:

__AQL explain (if applicable)__:

__Dataset__:

__Replication Factor & Number of Shards (Cluster only)__:

## Steps to reproduce

1. 
2. 
3. 

__Problem__:
```
2018-09-24T07:45:03Z [31243] ERROR got error from server: HTTP 500 (Internal Server Error):
 ArangoError 4: could not create index: internal error 
(exception location: /var/lib/otherjenkins/workspace/RELEASE__BuildPackages/arangod/RestHandler/RestReplicationHandler.cpp:850).
 Please report this error to arangodb.com
```

__Expected result__:

**Comment by ranjithk-redbus** [comment]
single server mode

**Comment by vinaypyati** [comment]
Hi @ranjithk-redbus,

Thanks for opening this issue. In order to analyze the problem could you describe bit more of context and provide us the following

- Which OS and storage-engine are you using?
- Possible steps that you carried out which led to this error.
- The dataset/dump that you are restoring.

Thanks!

**Comment by vinaypyati** [comment]
Hi @ranjithk-redbus,

Just a quick follow up.  How things are going? Did you still face this issue? 
Thanks!

Best,

**Comment by maxkernbach** [comment]
Since there was no feedback in a while, I am closing this issue for now.

Please feel free to reopen the issue by adding a new comment.

**Comment by vinaypyati** [comment]
Hi @gcallaghan,

Thanks for the detailed information. I am able to reproduce this issue with `auditedges.structure.json` that you provided on Windows 10 too - v3.3.19. 
```
usr\bin>arangorestore.exe --input-directory foo
Please specify a password:
Server version: 3.3.19
# Connected to ArangoDB 'http+tcp://127.0.0.1:8529'
# Re-creating edge collection 'auditedges'...
# Creating indexes for collection 'auditedges'...
2018-11-02T11:04:23Z [3576] ERROR got error from server: HTTP 500 (Internal Server Error): ArangoError 4: could not create index: internal error (exception location: c:\b\workspace\release__buildpackages\arangod\resthandler\restreplicationhandler.cpp:850). Please report this error to arangodb.com
Processed 1 collection(s), read 0 byte(s) from datafiles, sent 0 data batch(es)
```

**Comment by gcallaghan** [comment]
I was able to work around this using the following
```
arangorestore --server.endpoint=tcp://127.0.0.1:8528 --server.database=REDACTED --force --create-database=true --server.password=REDACTED --input-directory foo
Creating database 'SOME_DB'
```
Although I worry about the index error

**Comment by dothebart** [comment]
closing as solved with the availability of [ArangoDB 3.7](https://www.arangodb.com/2020/08/arangodb-3-7-a-big-step-forward-for-multi-model/) .
